legal-api/src/legal_api/services/authz.py

```diff
@@ -155,6 +155,13 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
                     'business': [BusinessBlocker.DEFAULT, BusinessBlocker.NOT_IN_GOOD_STANDING]
                 }
             },
+            'continuationOut': {
+                'legalTypes': ['BC', 'BEN', 'CC', 'ULC'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.NOT_IN_GOOD_STANDING],
+                    'completedFilings': ['consentContinuationOut']
```

Comment:
We need code to actually do the check for `completedFilings`.  Take a look at the code used for `validStateFilings`.  It won't be the same but the pattern will be similar I think.